oh_my_minishell/execute_exit.c

```diff
@@ -0,0 +1,97 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   execute_exit.c                                     :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: jikang <jikang@student.42seoul.kr>         +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2021/01/04 21:30:41 by jikang            #+#    #+#             */
+/*   Updated: 2021/01/06 19:29:31 by jikang           ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "minishell.h"
+
+/* 숫자만 있으면 1, 숫자 아닌 것도 있으면 0 */
+static int is_only_num(char *str)
+{
+	int i;
+
+	i = 0;
+	while (str[i] != '\0')
+	{
+		/* 숫자 아닌 것이 있으면 */
+		if (!ft_isdigit(str[i]))
+			return (0);
+		i++;
+	}
+	return (1);
+}
+
+static void one_argv(char *argv)
+{
+	/* 숫자만 있냐 아니냐? 판단하기 */
+	if (is_only_num(argv))
+	{
+		ft_putendl_fd("exit", 1);
+		exit((unsigned char)ft_atoi(argv));
+	}
+	else
+	{
+		ft_putendl_fd("exit", 1);
+		ft_putstr_fd("bash: exit: ", 1);
+		ft_putstr_fd(argv, 1);
+		ft_putendl_fd(": numeric argument required", 1);
+		exit(255);
+	}
+}
+
+static void more_than_one_argv(char *argv)
+{
+	/* 숫자만 있냐 아니냐? 판단하기 */
+	if (is_only_num(argv))
+	{
+		ft_putendl_fd("exit", 1);
+		ft_putendl_fd("bash: exit: too many arguments", 1);
+		exit(1);
```

Comment:
```suggestion
// 나중에 exit status를 저장할 $?에 1을 세팅해주는 거 추가
```
too many arguments 일때는 실제로 exit을 수행하지 않는 것같음
음
`$?` 이 `1`로 set되는건 맞는듯